API.md

```diff
@@ -85,7 +85,7 @@ let options = {
 
 `ignoreErrors` (default `true`) tells hapi that it should not respond with a HTTP 400 error if the session cookie cannot decrypt.  This could happen if the cookie is changed on the client, or more likely, if you change the cookie password in your settings.  If you want to make this condition send an error like it did in prior versions, change this to `false`, but be aware that if you change your cookie password you will cause 400 errors to be returned to end users.  In that case you should probably change this back to true for a short time to allow session cookies to get reset for the best user experience.
 
-You may turn this off, `false`, and try to use the hapi route state config option of `failAction` to instead get an event whenever a bad session cookie is encountered.  This can allow more sophisticated handling strategies or even allow for mitigation of brute force attacks on your cookie password.  See [server.state](http://hapi.dev/api#serverstatename-options) documentation for more details.
+You may turn this off, `false`, and try to use the hapi route state config option of `failAction` to instead get an event whenever a bad session cookie is encountered.  This can allow more sophisticated handling strategies or even allow for mitigation of brute force attacks on your cookie password.  See [server.state()](https://hapi.dev/api#-serverstatename-options) documentation for more details.
```

Comment:
```suggestion
You may turn this off, `false`, and try to use the hapi route state config option of `failAction` to instead get an event whenever a bad session cookie is encountered.  This can allow more sophisticated handling strategies or even allow for mitigation of brute force attacks on your cookie password.  See [server.state()](https://hapi.dev/api#server.state()) documentation for more details.
```